File: app/src/main/java/com/marcelorcorrea/falae/FalaeApplication.java
package com.marcelorcorrea.falae;

import android.app.Application;
import android.content.Intent;

import com.marcelorcorrea.falae.service.TextToSpeechService;

/**
 * Created by corream on 29/05/2017.
 */

public class FalaeApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        startService(new Intent(this, TextToSpeechService.class));
    }
}

File: app/src/main/java/com/marcelorcorrea/falae/FalaeApplication.kt
package com.marcelorcorrea.falae

import android.app.Application
import android.content.Intent

import com.marcelorcorrea.falae.service.TextToSpeechService

/**
 * Created by corream on 29/05/2017.
 */

class FalaeApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        startService(Intent(this, TextToSpeechService::class.java))
    }
}
